src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_transformers.py

```diff
@@ -46,3 +46,22 @@ def reservation_details_output(result):
 
 def transform_reservation_details_list_output(result):
     return [reservation_details_output(item) for item in result]
+
+
+def pricesheet_get_properties(result):
+    result.unit_of_measure = str(result.unit_of_measure)
+    result.included_quantity = str(result.included_quantity)
+    result.unit_price = str(result.unit_price)
+    if result.meter_details:
+        result.meter_details.total_included_quantity = str(result.meter_details.total_included_quantity)
+        result.meter_details.pretax_standard_rate = str(result.meter_details.pretax_standard_rate)
```

Comment:
Why is this necessary?  It looks like it is simply calling `str(...)` on every property.

Reply:
Im not sure; what I noticed from other functions here is that str() has been performed on all numeric values.

Reply:
What happens if the str() calls are removed?

Reply:
@williexu - Regarding str conversion, if removed, json conversion fails when reading decimal or any non-string type.